test/recipes/80-test_ca.t

```diff
@@ -71,6 +72,158 @@ SKIP: {
        "Signing SM2 certificate request");
 }
 
+test_revoke('notimes', {
+    should_succeed => 1,
+});
+test_revoke('lastupdate_invalid', {
+    lastupdate     => '1234567890',
+    should_succeed => 0,
+});
+test_revoke('lastupdate_utctime', {
+    lastupdate     => '200901123456Z',
+    should_succeed => 1,
+});
+test_revoke('lastupdate_generalizedtime', {
+    lastupdate     => '20990901123456Z',
+    should_succeed => 1,
+});
+test_revoke('nextupdate_invalid', {
+    nextupdate     => '1234567890',
+    should_succeed => 0,
+});
+test_revoke('nextupdate_utctime', {
+    nextupdate     => '200901123456Z',
+    should_succeed => 1,
+});
+test_revoke('nextupdate_generalizedtime', {
+    nextupdate     => '20990901123456Z',
+    should_succeed => 1,
+});
+test_revoke('both_utctime', {
+    lastupdate     => '200901123456Z',
+    nextupdate     => '200908123456Z',
```

Comment:
Is 2009 instead of 2099 intentional?

Reply:
Yes - those are two-digit-year timestamps (e.g. 2020-09-01 12:34:56), so OpenSSL should parse them as UTCTimes. The others are four-digit-year, post-2049 timestamps (e.g. 2099-09-01 12:34:56), so OpenSSL should (if I understood correctly) parse them as GeneralizedTimes. Does that make sense?